extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/view.html

```diff
@@ -3,26 +3,30 @@
 {#style}
 {/style}
 {#body}
-{#let folder=currentRequest.params.get("path")}
-<div class="row">
+ {#let folder=currentRequest.params.get("path")}
+  <div class="row">
     <div class="col">
-        {folder}
+      {folder}
     </div>
-</div>
-{#for childOfFolder in info:staticResourceInfo.resourceMap.get(folder).children}
-<div class="row">
-    <div class="col-1"></div>
-    <div class="col">
+  </div>
+  {#for childOfFolder in info:staticResourceInfo.resourceMap.get(folder).children}
+    <div class="row">
+      <div class="col-1"></div>
+      <div class="col">
         {#if !childOfFolder.isFolder}
-            <i class="fas fa-file-alt" style="margin-right:5px; margin-top:5px"></i>
+          <i class="fas fa-file-alt" style="margin-right:5px; margin-top:5px"></i>
+          <a href="/{childOfFolder.lastName}">
```

Comment:
All seems ok to me except link part in view.html `<a href="/{childOfFolder.lastName}">` , should we link here @FroMage ?

Reply:
I think it should be `{folder}/{childOfFolder.lastName}` no?

Reply:
Yes, done :)